src/server/api/errors.js

```diff
@@ -30,6 +30,21 @@ export async function accessRequired(
   }
 }
 
+export async function assignmentOrAdminRequired(
```

Comment:
Can we rename this to `assignmentOrAdminRoleRequired` -- so it's not confused with is_superadmin